requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy>=1.10.4,<1.20
+numpy>=1.10.4
```

Comment:
numpy 1.10.4 is from 2016, do we know it is working with this numpy? If not, possibly just remove the lower version limit to not "guarantee" anything.

Reply:
It  was from RMS 10 which use numpy 10.1. But I agree, will change to 1.13 (RMS 11)